airflow/ui/test/utils.tsx

```diff
@@ -0,0 +1,39 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { QueryClient, QueryClientProvider } from 'react-query';
+
+export const defaultHeaders = {
+  'access-control-allow-origin': '*',
```

Comment:
Guessing we'd want to change here too, or maybe it should pull from the `API_URL` value?
```suggestion
  'access-control-allow-origin': 'http://127.0.0.1:28080/api/v1/',
```